dev/vagrant/scripts/vagrant-ubuntu-deps.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+set -o errexit
+
+sudo apt-get update
+sudo DEBIAN_FRONTEND=noninteractive apt-get install -y unzip curl vim \
+    apt-transport-https \
+    ca-certificates \
+    software-properties-common
+
+sudo mkdir /etc/nomad
```

Comment:
This could be moved to the `vagrant-ubuntu-nomad-pkg.sh` script, so that all things related to installing Nomad would be in a single script, what do you think?